### PR TITLE
ex18_ch4.py

### PR DESCRIPTION
**Pull request by amymhaddad**
Instructions: Create a program that converts temperatures from Fahren- heit to Celsius or from Celsius to Fahrenheit. Prompt for the starting temperature. The program should prompt for the type of conversion and then perform the conversion.

### CODE REVIEW

File: ex18_ch4.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+"""
+
+import math 
+
+#step1: get inputs from user on the temperature and whether the temp is in C or F
+temp = float(input("Enter the temperature: "))
+temp_type = input("Is that temperature in Celsius or Fahrenheit? Enter C for Celsius or F for Fahrenheit: ")
+
+#step2: write first conditional to convert C to F
+if "C" in temp_type:
```
Comment (paulghaddad) [inline ex18_ch4.py]: `if temp_type == "C"`

File: ex18_ch4.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    celsius_to_fahrenheit = math.ceil(temp * 9 / 5) + 32
+    print(f"The temperature in Fahrenheit is {celsius_to_fahrenheit} degrees.")
+
+#step3: write second conditional to convert F to C
+elif "F" in temp_type:
```
Comment (paulghaddad) [inline ex18_ch4.py]: `if temp_type == "F"`

File: ex18_ch4.py [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+#step1: get inputs from user on the temperature and whether the temp is in C or F
+temp = float(input("Enter the temperature: "))
+temp_type = input("Is that temperature in Celsius or Fahrenheit? Enter C for Celsius or F for Fahrenheit: ")
+
+#step2: write first conditional to convert C to F
+if "C" in temp_type:
+    celsius_to_fahrenheit = math.ceil(temp * 9 / 5) + 32
+    print(f"The temperature in Fahrenheit is {celsius_to_fahrenheit} degrees.")
+
```
Comment (paulghaddad) [inline ex18_ch4.py]: No extra line needed.

File: ex18_ch4.py [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+#step3: write second conditional to convert F to C
+elif "F" in temp_type:
+    fahrenheit_to_celsius = math.ceil(temp - 32) * 5 / 9
+    print(f"The temperature in Celsius is {fahrenheit_to_celsius} degrees.")
```
Comment (paulghaddad) [inline ex18_ch4.py]: Great job on the calculation and the string formatting!

**Comment by amymhaddad** [comment]
@paulghaddad I made changes to this exercise.

**Pull request merged**